FM: Add from neo4j-admin CLI (#991)
neo4j/docs-aura: modules/ROOT/pages/fleet-management/setup.adoc

*Never register a token obtained from a third party.*
====

> == Discover deployments

Reviewer: ```suggestion
[[discover-deployments]]
== Discover deployments
```

Reviewer: I'll use this anchor to link to this section from the Operations Manual.

You can add deployments discovered on your network using a registration code provided by the Neo4j Admin CLI tool.
This involves first running the `neo4j-admin fleet discover` command to look for running Neo4j servers on your network.

> For more information on discovering running Neo4j servers with Neo4j Admin, see link:https://neo4j.com/docs/operations-manual/current/neo4j-admin-neo4j-cli/[Operations Manual -> Neo4j Admin and Neo4j CLI].

Reviewer: We need to add information to the Operations manual. 😄 Currently, we have only the command.

Reviewer: I'll add some.

Author: Agreed!

We also have the discovery broadcast feature going out with 2026.05: https://github.com/neo-technology/neo4j/pull/37297 Which is what the neo4j-admin command uses to discover the servers.

I was thinking it would make sense to document both that and this on a new page in the ops manual.

Reviewer: Sounds good.

> After you have obtained a registration code from the Neo4j Admin CLI tool, select *Discover deployments* which can be found by selecting the *down arrow* next to the *Add deployment* button [See Figure 1]. In the drawer that appears, paste the code in the indicated input box.

Reviewer 2: ```suggestion
After you have obtained a registration code from the Neo4j Admin CLI tool, select *Discover deployments* which can be found by using the *down arrow* next to the *Add deployment* button [See Figure 1]. 
Paste the code in the indicated input box.
```

[.shadow]
.Add a new self-managed deployment
image::add-from-cli.png[width=600]

> The list of deployments discovered by `neo4j-admin` appears in the drawer. Next to each deployment is a *Register* button which can be used to register each deployment in turn. Select this button to initiate the setup wizard which is described in previous sections.

Reviewer 2: ```suggestion
The list of deployments discovered by `neo4j-admin` appears in the drawer. 
Next to each deployment is a *Register* button which can be used to register each deployment in turn. 
Use this button to initiate the setup wizard which is described in previous sections.
```

*Never register a token obtained from a third party.*
====

[[discover-deployments]]
== Discover deployments

You can add deployments discovered on your network using a registration code provided by the Neo4j Admin CLI tool.
This involves first running the `neo4j-admin fleet discover` command to look for running Neo4j servers on your network.

> For more information on discovering running Neo4j servers with Neo4j Admin, see link:https://neo4j.com/docs/operations-manual/current/neo4j-admin-neo4j-cli/[Operations Manual -> Neo4j Admin and Neo4j CLI].

Reviewer: ```suggestion
For more information on discovering running Neo4j servers with Neo4j Admin, see link:https://neo4j.com/docs/operations-manual/current/monitoring/fleet-discovery/[Operations Manual -> Discover local servers].
```

After you have obtained a registration code from the Neo4j Admin CLI tool, select *Discover deployments* which can be found by using the *down arrow* next to the *Add deployment* button [See Figure 1]. 
Paste the code in the indicated input box.

[.shadow]
.Add a new self-managed deployment
image::add-from-cli.png[width=600]

The list of deployments discovered by `neo4j-admin` appears in the drawer. 

> Next to each deployment is a *Register* button which can be used to register each deployment in turn. 

Reviewer: ```suggestion
Next to each deployment is a *Register* button that can be used to register it. 
```

> Use this button to initiate the setup wizard which is described in previous sections.

Reviewer: ```suggestion
Use this button to start the setup wizard described in the previous sections.
```
